pages/about.html

```diff
@@ -0,0 +1,129 @@
+				</ul>
+				<nav class="desktop-nav-items">
+					<a href="/" class="wd-logo">WD<span>Course</span></a>
+					<div class="menu-cc">
+						<ul class="desktop-nav-items-links">
+							<li class="desktop-nav-item-link about"><a href="/">Home</a></li>
+							<li class="desktop-nav-item-link"><a href="#">Program</a></li>
+							<li class="desktop-nav-item-link"><a href="#">Join</a></li>
+							<li class="desktop-nav-item-link"><a href="#">Sponsor</a></li>
+							<li class="desktop-nav-item-link"><a href="#">News</a></li>
+						</ul>
+						<a class="desktop-cc">CC Campaign</a>
+					</div>
+				</nav>
+			</nav>
+		</header>
+
+		<main>
+			<section id="about-hero" class="about-hero">
+				<div class="about-heading">
+					<p class="about-hello">"Hello! Learner world"</p>
+					<h2>Build websites and applications with the best</h2>
+					<div class="trait"></div>
+				</div>
+				<div class="about-desc">
+					<p>Each aspect of creating websites and applications entails a unique set of skills. Udemy offers a host of
+						courses to bring you up to speed on modern front-end, back-end, and fullstack web development practices and
+						skills.</p>
+				</div>
+				<div class="about-small">
+					<small>Please contact us per Email for any further questions about Build Website and Applications! <br>
+						<strong>jeanemmanuelcadet@gmail.com</strong></small>
+				</div>
+			</section>
+
+			<section id="about-logo" class="about-logo">
+				<div class="about-logo-heading">
+					<h1>Build websites and applications with the best</h1>
+					<div class="trait"></div>
+					<p>The logo of Build websites and applications with the best was decided through the logo competition</p>
+					<div class="about-logo-img">
+						<a href="/" class="wd-logo">WD<span>Course</span></a>
+					</div>
+				</div>
+			</section>
+			<hr class="about-hr">
+
+			<section id="see-past" class="about-logo">
+				<div class="see-past-heading">
+					<h1>See the past CC Global Summits</h1>
+					<div class="trait"></div>
+					<p>Take a look at the last two CC Global Summits which took place in Buenos Aires and in Warsaw.</p>
+					<div class="see-img">
+						<img src="../images/about-2013.png" alt="Photo about 2013">
+						<img src="../images/about-2011.png" alt="Photo about 2011">
+					</div>
+
+				</div>
+			</section>
+
+			<section id="partners" class="partners">
+				<div class="partners-heading">
+					<h1>Partner</h1>
+					<div class="trait"></div>
+				</div>
+				<div class="partner-contents">
+					<img src="../images/mozilla-logo.png" alt="Mozilla Logo">
+					<img src="../images/google-logo.png" alt="Google Logo">
+					<img src="../images/naver-logo.png" alt="Naver Logo">
+					<small>daumkakao</small>
+					<img src="../images/airbnb-logo.png" alt="Airbnb Logo">
+				</div>
+			</section>
+
+			<footer>
+				<a href="/" class="wd-logo">WD<span>Course</span></a>
+				<p>2023 Jecode93 © Copyright All right reserved</p>
+			</footer>
```

Comment:
- [ ] According to the design guidelines, the footer of your About page should have a dark background. please consider applying it! 🤓 🔧 

| Template |
|---|
| ![image](https://github.com/jecode93/first-module-capstone-project/assets/77069011/a7cface0-52e5-4b43-9bb0-615acfab3efa) |
| Your Design |
| ![image](https://github.com/jecode93/first-module-capstone-project/assets/77069011/aa7d0df0-c2ef-4dee-8e22-f420b4a3e1d7) |